File: WebApplication2/Dtos/YogaLessonDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using YogaStudio.Models;

namespace YogaStudio.Dtos
{
    public class YogaLessonDto
    {
        public int Id { get; set; }
        public DayOfWeek Day { get; set; }
        public string StartingTime { get; set; }
        public string EndTime { get; set; }
        public Level ClassLevel { get; set; }

        public List<string> studentsIds { get; set; } = new List<string>();


    }
}

File: YogaClient/YogaStudio/src/app/Models/YogaLessonDto.ts
import { StudentInClass } from "./StudentInClass";

export class YogaLessonDto{
    id: number|null;
    day: number;
    startingTime: any;
    endTime: any;
    classLevel: number;
    studentsIds: number[]=[];

    /**
     *
     */
    constructor(day,startingTime,endTime) {
        this.day=day;
        this.startingTime=startingTime;
        this.endTime=endTime;
      
        
    }
}